core/controllers/custom_landing_pages.py

```diff
@@ -30,22 +29,26 @@ class FractionLandingRedirectPage(base.BaseHandler):
     @acl_decorators.open_access
     def get(self):
         """Handles GET requests."""
-        self.redirect('/learn/maths/fractions')
+        self.redirect('/math/fractions')
+
+
+class TopicRedirectPage(base.BaseHandler):
+    """The handler redirecting the old landing page URL to the new one."""
+
+    @acl_decorators.open_access
```

Comment:
@vojtechjelinek, Can we statically redirect uses to a different pages (through app.yaml?)

Reply:
(I realized that we are generating different URLs for this handler in python, so I don't think we can still do this)